client/app/telecom/telephony/billingAccount/manage/telecom-telephony-billing-account-manage.controller.js

```diff
@@ -1,60 +0,0 @@
-angular.module("managerApp").controller("TelecomTelephonyBillingAccountManageCtrl", function ($translate, $stateParams, TelephonyMediator, ToastError) {
```

Comment:
`client/app/telecom/telephony/billingAccount/manage` isn't used anymore then I decided to remove it.